scripts/nouns/zkvote_contract.ts

```diff
@@ -0,0 +1,56 @@
+import {
+    ZKVote, ZKVote__factory, NvoteVerifier__factory,
+    TallyVerifier__factory,
+  } from "../types";
+  import { Signer, Contract } from "ethers";
+  import { Provider } from "@ethersproject/providers";
+  export { ZKVote } from "../types";
+  
+  
+  export type ZKVoteContractDescriptor = {
```

Comment:
Doesn't have to be done now, but I think we can probably wind the dkg deployment into here, deploy both at the same time, and include the dkg contract address as another field in the descriptor.

Again, no need to do it now - we can just add it as an issue for later if you think it makes sense.

Reply:
Will keep this for later, as suggested.